deployments.md

```diff
@@ -23,18 +16,14 @@ They can be published at any time if they are backwards compatible with the last
 Breaking changes should be deployed at the same time as a major compiler release.
 
  1. Run `yarn install`.
- 2. Run `node_modules/.bin/lerna version`.
-    The command will ask you to choose a new version number.
+ 2. Run `yarn version (patch|minor)`.
```

Comment:
No the intent is that the CI platform runs the publish and its the only one building the artifacts and with the publication token. I try to avoid publishing directly from my own machine.